README.md

```diff
@@ -251,15 +228,27 @@ For launch `foggerid` can be obtained from the miner's account:
 
 Choose `nodeid` for for future identification of this machine. It should be unique alphanumeric value without spaces.
 
-Fog Agent will collect the information about your node and register it in your account
+Starting with version 0.7.0, FR Node is part of Fog Agent. Now Faceter FOG Client is a single component. 
+Fog Agent is installing, running and updating FR Node.
+
+Each action of Fog Agent is written to the terminal and to the log files located along the path `/var/log/fogagent/{Date}.log`, where {Date} is the date the logs were written in the format `20200420`. 
```

Comment:
All Fog Agent's actions should be written into the terminal (non-service running mode) and to log files, located at
'/var/log/fogagnet/yyyyMMdd.log'